src/service/user.service.js

```diff
@@ -0,0 +1,65 @@
+import { ApiError } from '../exeptions/api.error.js';
+import { User } from '../moduls/User.js';
+import { Console } from '../untils/Console.js';
+
+function validateEmail(value) {
+  const EMAIL_PATTERN = /^[\w.+-]+@([\w-]+\.){1,3}[\w-]{2,}$/;
+
+  if (!value) {
+    return 'Email is required';
+  }
+
+  if (!EMAIL_PATTERN.test(value)) {
+    return 'Email is not valid';
+  }
+}
+
+function validatePassword(value) {
+  if (!value) {
+    return 'Password is required';
+  }
+
+  if (value.length < 6) {
+    return 'At least 6 characters';
+  }
+}
+
+async function validUserName(value) {
+  const existUser = await User.findOne({ where: { value } });
```

Comment:
The `validUserName` function is checking for a user with a `value` field, which might be incorrect. Ensure that you are checking the correct field, likely `userName`.